ProfilesMicroService.Infrastructure/ApplicationDBContext.cs

```diff
@@ -0,0 +1,49 @@
+﻿using Microsoft.EntityFrameworkCore;
+using ProfilesMicroService.Api.DataBaseConfiguration;
+using ProfilesMicroService.Domain.Entities.Models;
+
+namespace ProfilesMicroService.Infrastructure
+{
+    public class ApplicationDBContext : DbContext
+    {
+        public DbSet<Doctor> Doctors { get; set; }
+        public DbSet<Receptionist> Receptionists { get; set; }
+        public DbSet<Patient> Patients { get; set; }
+
+        public DbSet<Status> Statuss { get; set; }
+
+        public ApplicationDBContext(DbContextOptions<ApplicationDBContext> options) : base(options) { }
+
+        protected override void OnModelCreating(ModelBuilder builder)
+        {
+            base.OnModelCreating(builder);
+            builder.Entity<Receptionist>().Property(e => e.Id).HasDefaultValueSql("gen_random_uuid()");
```

Comment:
I think it's redudant, Id of type Guid is automatically added as default behavior of ef

Reply:
postgresql don't allow you to generate guid without this string